contracts/PrecedenceCampaignArbitrable.sol

```diff
@@ -0,0 +1,76 @@
+pragma solidity ^0.5.8;
+
+import "@aragon/court/contracts/arbitration/IArbitrable.sol";
+import "@aragon/court/contracts/arbitration/IArbitrator.sol";
+
+
+contract PrecedenceCampaignArbitrable is IArbitrable {
+    //bytes4 public constant ERC165_INTERFACE = ERC165_INTERFACE_ID;
+    //bytes4 public constant ARBITRABLE_INTERFACE = ARBITRABLE_INTERFACE_ID;
+    string public constant ERROR_NOT_ALLOWED = "PCA_NOT_ALLOWED";
+
+    address public owner;
+    IArbitrator public arbitrator;
+
+    modifier only(address _who) {
+        require(msg.sender == _who, ERROR_NOT_ALLOWED);
+        _;
+    }
+
+    constructor (address _owner, IArbitrator _arbitrator) public {
+        owner = _owner;
+        arbitrator = _arbitrator;
+    }
+
+    function createDispute(uint256 _possibleRulings, bytes calldata _metadata) external only(owner) returns (uint256) {
+        return _createDispute(_possibleRulings, _metadata);
+    }
+
+    function submitEvidence(uint256 _disputeId, bytes calldata _evidence, bool _finished) external only(owner) {
+        _submitEvidence(_disputeId, msg.sender, _evidence, _finished);
+    }
+
+    function submitEvidenceFor(uint256 _disputeId, address _submitter, bytes calldata _evidence, bool _finished) external only(owner) {
+        _submitEvidence(_disputeId, _submitter, _evidence, _finished);
+    }
+
+    function createAndSubmit(
+        uint256 _possibleRulings,
+        bytes calldata _metadata,
+        address _submitter1,
+        address _submitter2,
+        bytes calldata _evidence1,
+        bytes calldata _evidence2
```

Comment:
We can probably extend this version to use a list of submitters and evidence, if it reverts due to an OOG is not harmful

Reply:
It gives me the following error:
```
TypeError: This type is only supported in the new experimental ABI encoder. Use "pragma experimental ABIEncoderV2;" to enable the feature.
```
So I don't think it's worth the hassle.